### ISSUE TITLE
Understanding HPP

### ISSUE THREAD

**Issue by teguhSL** [humanoid-path-planner/hpp_tutorial#15]
Hi, I have tried the hpp_tutorial (tutorial 1 and 2). Could anyone suggest on how to proceed, in order to understand how to use HPP better? To give a context, I am interested in using HPP to plan motion for a humanoid, while considering collision checking, balance, etc. 

I'd like to understand a few things in HPP:

-     How to add another robot or a new environment
-     How to use another solver. The tutorial 2 shows this, but only by specifying directly "PRM" (which I think has been included in hpp). If I want to use my own solver, would it be easy with HPP?
-     Can we Initialise the solver with a particular solution (trajectory)?
-     Can we define the set of active joints to be optimized by the solver

so basically to understand further about setting up the environments, the robots, the optimization problem, and the solver. Thank you!

**Comment by jmirabel** [comment]
For reference, this follows https://github.com/Gepetto/gepetto-viewer-corba/issues/87.

**Comment by jmirabel** [comment]
> On tutorial 3, I get the following error, and the hpp-manipulation-server gives me segmentation fault.

On the devel branch, I have no SEGV. I'll try the stable version.

> How to add another robot or a new environment

Have a look at `hpp_tutorial/src/hpp/corbaserver/pr2/robot.py`. This file is included in tutorial 1 with `from hpp.corbaserver.pr2 import Robot`. For the manipulation stack, see `hpp_tutorial/src/hpp/corbaserver/manipulation/pr2/robot.py`

For the environment, check the documentation of the functions used in tutorial 1 and 3 to load the obstacles. Use the search bar in the doc of hpp_tutorial (link 'tutorial' on the left side of the doc).

**Comment by jmirabel** [comment]
From Gepetto/gepetto-viewer-corba#87
> On tutorial 3, I get the following error, and the hpp-manipulation-server gives me segmentation fault.

I recompiled everything from scratch and ran tutorial 3. It worked fine on my computer.

**Comment by florent-lamiraux** [comment]
I close this issue. Please feel free to reopen it.
